core/arch/arm/kernel/secure_partition.c

```diff
@@ -410,11 +400,10 @@ static TEE_Result sp_dt_get_uuid(const void *fdt, int node,
 	return TEE_SUCCESS;
 }
 
-static TEE_Result check_fdt(const void * const fdt, const TEE_UUID *uuid)
+static TEE_Result check_fdt(const void * const fdt, TEE_UUID *uuid)
```

Comment:
This function does a bit more than "checking" the fdt now and should preferrably have a name to reflect that.

Reply:
Ack